legend-engine-xt-serviceStore-protocol/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/packageableElement/store/service/model/Service.java

```diff
@@ -22,5 +23,5 @@ public class Service extends ServiceStoreElement
     public HttpMethod method;
     public List<ServiceParameter> parameters;
     public ComplexTypeReference response;
-    public List<SecurityScheme> security;
+    public List<String> security = new ArrayList<>();
```

Comment:
This seems like a backward incompatible change? We should avoid this change and mark the field deprecated. Also no assignment in protocol classes
Also do we not need securityRequirement property I saw you made that change in protocol already?